05week/transitions-and-transformations/css/style.css

```diff
@@ -0,0 +1,84 @@
+body {
+  margin: 0 10px;
+}
+
+.container {
+  margin: 0 650px;
```

Comment:
you could have used margin:auto or flex box on the parent to center the container